libseccomp-sys/src/lib.rs

```diff
@@ -638,6 +646,29 @@ extern "C" {
     /// This function generates seccomp Berkeley Packer Filter (BPF) code and writes
     /// it to the given fd.  Returns zero on success, negative values on failure.
     pub fn seccomp_export_bpf(ctx: const_scmp_filter_ctx, fd: c_int) -> c_int;
+
+    /// Generate seccomp Berkeley Packet Filter (BPF) code and export it to a buffer
+    ///
+    /// - `ctx`: the filter context
+    /// - `buf`: the destination buffer
+    /// - `len`: on input the length of the buffer, on output the number of bytes in the program
+    ///
+    /// This function generates seccomp Berkeley Packer Filter (BPF) code and writes
+    /// it to the given buffer.  Returns zero on success, negative values on failure.
+    pub fn seccomp_export_bpf_mem(
+        ctx: const_scmp_filter_ctx,
+        buf: *mut c_void,
+        len: *mut size_t,
+    ) -> c_int;
+
+    ///  Precompute the seccomp filter for future use
+    ///
+    ///  - `ctx`: the filter context
+    ///
+    ///  This function precomputes the seccomp filter and stores it internally for
+    ///  future use, speeding up [`seccomp_load()`] and other functions which require
+    ///  the generated filter.
+    pub fn seccomp_precompute(ctx: scmp_filter_ctx) -> c_int;
```

Comment:
In seccomp.h it is defined with `**const** scmp_filter_ctx ctx`.

Reply:
fixed